pkg/apis/monitoring/v1/alertmanager_types.go

```diff
@@ -206,6 +206,9 @@ type AlertmanagerSpec struct {
 	ClusterAdvertiseAddress string `json:"clusterAdvertiseAddress,omitempty"`
 	// Interval between gossip attempts.
 	ClusterGossipInterval GoDuration `json:"clusterGossipInterval,omitempty"`
+	// Defines the identifier that uniquely identifies the Alertmanager cluster.
+	// You should only set it when the Alertmanager cluster includes more than one Alertmanager replicas.
```

Comment:
That's not entirely exact. Even with multiple Alertmanager pods, you shouldn't need to set this field as long as you don't join pods across different clusters.

Reply:
@ArthurSens this was your suggestion, maybe you can chime in?

Reply:
sorry my initial suggestion wasn't exactly what I had in mind. Refining here.
```suggestion
	// You should only set it when the Alertmanager cluster includes Alertmanager instances which are external to this Alertmanager resource. In practice, the addressed of the external instances are provided via the `.spec.additionalPeers` field.
```

Reply:
This describes it way better, you should not worry about it, if you don't have external AMs joining the cluster. Wasn't sure about how extensive this description should be here.
Would take it this way and re-render the package.

Reply:
> @ArthurSens this was your suggestion, maybe you can chime in?

Yeah, sorry for the shallow review, I did not look in depth at the problem. What Simon suggested looks a lot more precise